src/ts_catalog/continuous_agg.c

```diff
@@ -426,12 +427,11 @@ continuous_agg_fill_bucket_function(int32 mat_hypertable_id, ContinuousAggsBucke
 		 */
 		heap_deform_tuple(tuple, ts_scan_iterator_tupledesc(&iterator), values, isnull);
 
-		Assert(!isnull[Anum_continuous_aggs_bucket_function_experimental - 1]);
-		bf->experimental =
-			DatumGetBool(values[Anum_continuous_aggs_bucket_function_experimental - 1]);
+		Assert(!isnull[Anum_continuous_aggs_bucket_function_function - 1]);
```

Comment:
```suggestion
		Assert(!isnull[AttrNumberGetAttrOffset(Anum_continuous_aggs_bucket_function_function)]);
```

Reply:
I've sent a separated PR adding a coccinele script to check this mistake. https://github.com/timescale/timescaledb/pull/6661

Reply:
I noticed this too. I had this refactoring on my toto-list for a follow-up PR. Thanks for implementing the coccinelle rule! 

I changed this specific line to `AttrNumberGetAttrOffset` and we should merge your PR before.